neurom/view/view.py

```diff
@@ -182,10 +202,13 @@ def plot_neuron(ax, nrn,
         linewidth(float): all segments are plotted with this width, but only if diameter_scale=None
         color(str or None): Color of plotted values, None corresponds to default choice
         alpha(float): Transparency of plotted values
-    """
+        scale_diameter_with(str): axis along which to scale diameter for
+                                  correct unit (None, 'x' or 'y')
```

Comment:
Is there a way to have the scaling be always correct without having to provide this additional argument ?